astropy/utils/misc.py

```diff
@@ -224,7 +224,7 @@ def find_api_page(obj, version=None, openinbrowser=True, timeout=None):
     elif version == 'dev' or version == 'latest':
         baseurl = 'http://devdocs.astropy.org/'
```

Comment:
HTTPS does not work for this one, so I left it as-is.

Reply:
Hmm, that itself would seem a bug too. But to what? @bsipocz, @astrofrog - where is `devdocs.astropy.org` defined? Any idea why it doesn't support `https`?

Reply:
Let's summon @eteq

Reply:
but `astropy.org` supports https for a while now, doesn't it, so why this isn't working?

Reply:
Or... I can just change all the URLs to use RTD directly, since it redirects there anyway and RTD is all HTTPS. :woman_shrugging:  Otherwise, I have no idea...